apps/vast/vast-bridge.py

```diff
@@ -568,6 +615,14 @@ def main():
         default=None,
         help="If sink is specified, sightings are not reported back to Threat Bus. Instead, the context of a sighting (only the contents without the Threat Bus specific sighting structure) is forwarded to the specified sink via a UNIX pipe. This option takes a command line string to use and invokes it as direct subprocess without shell / globbing support.",
     )
+    parser.add_argument(
+        "--ulimit",
+        "-U",
+        dest="ulimit",
+        default=100,
```

Comment:
The same default was hardcoded above. Can we extract that value into a constant?

Reply:
I removed the default value from the `start` method so we only use this default.